### ISSUE TITLE
Pester tests throw error when using version 4 or higher

### ISSUE THREAD

**Issue by ergo3114** [PowerShellOrg/Plaster#416]
https://pester.dev/docs/migrations/v3-to-v4

**Comment by ergo3114** [comment]
Does not show error when using pwsh but does when using PowerShell Integrated Console.

### PowerShell Version in Integrated Console
```powershell
Name                           Value
----                           -----
PSVersion                      7.1.4
PSEdition                      Core 
GitCommitId                    7.1.4
OS                             Microsoft Windows 10.0.19043
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### pwsh
```powershell
Name                           Value
----                           -----
PSVersion                      7.1.4
PSEdition                      Core
GitCommitId                    7.1.4
OS                             Microsoft Windows 10.0.19043
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Pester
```powershell
Major  Minor  Build  Revision
-----  -----  -----  --------
5      2      2      -1
```

**Comment by ergo3114** [comment]
The error seems spotty. An example is below:

```text
[-] Module Error Handling Tests.Not supported schemaVersion.Throws on schemaVersion greater than latest supported schemaVersion 15ms (14ms|1ms)
 at <ScriptBlock>, C:\git\Plaster\test\PlasterManifestValidation.Tests.ps1:205
 RuntimeException: Legacy Should syntax (without dashes) is not supported in Pester 5. Please refer to migration guide at: https://pester.dev/docs/migrations/v3-to-v4
 RuntimeException: Legacy Should syntax (without dashes) is not supported in Pester 5. Please refer to migration guide at: https://pester.dev/docs/migrations/v3-to-v4
 ParameterBindingException: Cannot retrieve the dynamic parameters for the cmdlet. Legacy Should syntax (without dashes) is not supported in Pester 5. Please refer to migration guide at: https://pester.dev/docs/migrations/v3-to-v4
```

**Comment by ergo3114** [comment]
Resolved in #363
